frontend/language/src/nb.json

```diff
@@ -836,6 +836,8 @@
   "process_editor.configuration_panel_set_data_model_label": "Datamodell:",
   "process_editor.configuration_panel_set_data_model_link": "Legg til datamodell",
   "process_editor.configuration_panel_set_data_types_to_sign": "Datatyper som skal signeres:",
+  "process_editor.configuration_panel_set_unique_from_signatures_in_data_types": "Samme person kan ikke signere både denne oppgaven og:",
```

Comment:
also suggistion:
I wonder if we add more info about (`denne oppgaven`) like `(denne oppgaven du er på/ denne oppgaven du signerer nå)`

Reply:
I suggest we keep the text here as it has already been reviewed by Gørild. I also notice that we use `denne oppgaven` for other texts:

https://github.com/Altinn/altinn-studio/blob/382d8cfb2b355014b56d109a5296442df6f95f21/frontend/language/src/nb.json#L771

https://github.com/Altinn/altinn-studio/blob/382d8cfb2b355014b56d109a5296442df6f95f21/frontend/language/src/nb.json#L840